R/fftrees_create.R

```diff
@@ -462,13 +481,15 @@ fftrees_create <- function(data = NULL,
         acc = NA,
         bacc = NA, cost = NA, cost_decisions = NA, cost_cues = NA
       ),
+
       models = list(lr = NULL, cart = NULL, rf = NULL, svm = NULL)
     )
   )
 
-  class(x) <- "FFTrees"
+  class(x) <- "FFTrees"  # assign class
```

Comment:
Is this comment needed? I think it's pretty clear what the code means.

Reply:
I agree (and removed in next commit).